sonic-xcvrd/tests/optics_si_settings.json

```diff
@@ -0,0 +1,222 @@
+{
+   "GLOBAL_MEDIA_SETTINGS":{
+      "0-31":{
+         "100G_SPEED":{
```

Comment:
This '100G_SPEED' you are referring to here is the port speed or the lane speed?

Reply:
It refers to lane speed